oidc-controller/api/routers/presentation_request.py

```diff
@@ -104,5 +104,5 @@ async def send_connectionless_proof_req(
             service=s_d,
         )
         msg_contents = msg
-    print(msg_contents.dict(by_alias=True))
+    logger.info(msg_contents.dict(by_alias=True))
```

Comment:
Wondering if this should be a `debug` statement to avoid over-printing to logs when not necessary.